### PR TITLE
Fix bug for extension

### LINKED ISSUE #23: Fix the bug where the router does not work properly in the extension

### PR DESCRIPTION
**Pull request by HyeockJinKim**
Fixes #23

### CODE REVIEW

File: manifest.json [hunk 1]
```diff
@@ -13,6 +13,6 @@
   "icons": {
     "128": "public/icon.png"
   },
-  "content_security_policy": "script-src 'self' 'sha256-qILRdnleFFB8nUofQbeggst+MRykHHDGiqtfJvKdtPY='",
+  "content_security_policy": "script-src 'self' 'sha256-/U7yBM5I9QXyu5e9HHzfWjRJaVOkFHyM2S057j5ackU='",
```
Comment (HyeockJinKim) [inline manifest.json:16]: for extension csp policy

File: src/components/login/index.tsx [hunk 2]
```diff
@@ -51,6 +52,15 @@ const useStyles = makeStyles(() => ({
 
 // TODO: specify properties and their types
 const LoginPage = observer(({path}: LoginPageProps) => {
+  const history = useHistory();
+  const login = () => {
+    if (loginStore.login()) {
+      // TODO: should move prev page
+      history.push('/');
```
Comment (HyeockJinKim) [inline src/components/login/index.tsx:59]: when login is successful, move page

File: src/stores/login/index.ts [hunk 3]
```diff
@@ -18,7 +13,10 @@ class Login {
   }
 
   init() {
-    chrome.storage.local.get(['hash'], async data => {
+    if (chrome.storage === undefined) {
+      return;
+    }
```
Comment (HyeockJinKim) [inline src/stores/login/index.ts:18]: for test in react app, check chrome.storage.

**Review by HyeockJinKim — COMMENTED** [review]
self review

**Review by anthonyminyungi — APPROVED** [review]
Looks nice.

**Pull request merged**